code/__defines/skills.dm

```diff
@@ -12,25 +12,14 @@
 #define SKILL_AVERAGE  2
 #define SKILL_HARD     4
 
-#define SKILL_BUREAUCRACY   /decl/hierarchy/skill/organizational/bureaucracy
-#define SKILL_FINANCE       /decl/hierarchy/skill/organizational/finance
 #define SKILL_EVA           /decl/hierarchy/skill/general/EVA
-#define SKILL_MECH          /decl/hierarchy/skill/general/mech
```

Comment:
Does deadspace have mechas? I've not heard anything about this feature being removed, request input from west

Reply:
Dead Space does not have mecha.